tests/testthat/test-orderly.R

```diff
@@ -112,6 +112,8 @@ test_that("orderly_run_info: is_latest detects latest version", {
 
 
 test_that("orderly_test_start failure resets working directory", {
+  # if logging is on this test will wait for user input, so turn off
+  orderly_log_off()
```

Comment:
when changing global state in tests, use `on.exit` to restore

It might be better though to have a further option to disable package installation too (`orderly.no_install_missing`)